### ISSUE TITLE
doesnt work at all

### ISSUE THREAD

**Issue by RichPrune** [Tschipp/CarryOn#536]
### Expected Behavior
can carry things

### Actual Behavior
cant carry things

### Steps to Reproduce
shift right click a block (binding doesnt change this)
opens the entity like normal right click if it can, if it cant doesn't do anything

### Version of Minecraft, Carry On, Forge/Fabric
1.19.2 forge

### Screenshots encouraged
![2023-07-26_13 46 00](https://github.com/Tschipp/CarryOn/assets/103792572/95e05a29-a8ae-491c-a580-11083e65fba9)
[latest.log](https://github.com/Tschipp/CarryOn/files/12175845/latest.log)

**Comment by RichPrune** [comment]
just to clarify your mod doesn't work at all for anything; blocks, entities, etc; i reset the config and removed all other shift/right click keybinds and everything

**Comment by RichPrune** [comment]
also ive noticed you don't respond to other issues and just label them, could you please actually respond to this one?

**Comment by Tschipp** [comment]
I respond when needed. Clearly there is some interaction with another mod that is the issue, because with just carry on, it works. So I'd need to ask you to try to isolate which mod is causing the conflict.

**Comment by Chad90b** [comment]
@RichPrune this may sound silly, but please double-check your /mods/ folder and ensure you have the carryon-**forge**-[...] version. if you are installing mods via CurseForge you are close to guaranteed to have received the fabric version in error. They (CF) are not very good at this so you got to keep an eye out yourself too.

**Comment by Tschipp** [comment]
Yeah, good point. I'm gonna leave this open for now but close if nothing changes.
